deploy-manage/autoscaling/trained-model-autoscaling.md

```diff
@@ -2,28 +2,188 @@
+
+If the new allocations fit on the current {{ml}} nodes, they are immediately started. If more resource capacity is needed for creating new model allocations, then your {{ml}} node will be scaled up if {{ml}} autoscaling is enabled to provide enough resources for the new allocation. The number of model allocations can be scaled down to 0. They cannot be scaled up to more than 32 allocations, unless you explicitly set the maximum number of allocations to more. Adaptive allocations must be set up independently for each deployment and [{{infer}} endpoint](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-inference-put).
+
+:::{note}
+When you create inference endpoints on Serverless using Kibana, adaptive allocations are automatically turned on, and there is no option to disable them.
+:::
+
+### Optimizing for typical use cases [optimizing-for-typical-use-cases]
+You can optimize your model deployment for typical use cases, such as search and ingest. When you optimize for ingest, the throughput will be higher, which increases the number of {{infer}} requests that can be performed in parallel. When you optimize for search, the latency will be lower during search processes.
+
+* If you want to optimize for ingest, set the number of threads to `1` (`"threads_per_allocation": 1`).
+* If you want to optimize for search, set the number of threads to greater than `1`. Increasing the number of threads will make the search processes more performant.
+
+## Enabling autoscaling in {{kib}} - adaptive resources [enabling-autoscaling-in-kibana-adaptive-resources]
+$$$nlp-model-adaptive-resources$$$
+
+You can enable adaptive resources for your models when starting or updating the model deployment. Adaptive resources make it possible for {{es}} to scale up or down the available resources based on the load on the process. This can help you to manage performance and cost more easily. When adaptive resources are enabled, the number of vCPUs that the model deployment uses is set automatically based on the current load. When the load is high, the number of vCPUs that the process can use is automatically increased. When the load is low, the number of vCPUs that the process can use is automatically decreased.
+
+You can choose from three levels of resource usage for your trained model deployment; autoscaling will occur within the selected level’s range.
+
+Refer to the tables in the [Model deployment resource matrix](#model-deployment-resource-matrix) section to find out the setings for the level you selected.
+
+:::{image} ../../images/machine-learning-ml-nlp-deployment-id-elser-v2.png
+:alt: ELSER deployment with adaptive resources enabled.
+:class: screenshot
+:::
```

Comment:
this picture is really big and should be [sized down](https://elastic.github.io/docs-builder/syntax/images/#block-level-images)